protocol/xmrmaker/swap_state.go

```diff
@@ -334,6 +334,7 @@ func (s *swapState) tryClaim() (ethcommon.Hash, error) {
 	}
 
 	if untilT0 > 0 && stage != swapfactory.StageReady {
+		// TODO: t0 could be 24 hours from now. Don't we want to poll the stage periodically?
```

Comment:
@noot Fixing this is probably out of scope for this PR, but do you agree that this is a important issue to fix?

Reply:
yeah, this should probably be fixed. in the ideal case, the eth-provider should notify us when they call ready, but we can't necessarily count on that, so would be good to poll it here.